BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -198,8 +199,11 @@ open class BraintreeClient @VisibleForTesting internal constructor(
                             eventName,
                             params.payPalContextId,
                             params.linkType,
-                            venmoInstalled = isVenmoInstalled,
-                            isVaultRequest = params.isVaultRequest
+                            isVenmoInstalled,
```

Comment:
General style question for @saperi22 and @sshropshire, do we prefer the `param = param` syntax for Kotlin in v5 or just `param`?

Reply:
In Android, using `param = param` is necessary to specify the name of the parameter we are assigning a value to. Previously, between `linkType` and `venmoInstalled`, we had a parameter called `timestamp`, which had a default value. I moved this parameter to the end. Now, since all other parameters require a value assignment, it is no longer necessary to specify the name and value

Reply:
But we can still use it if we want correct? I was just curious if we had a style guide established yet for a preference since I think we are currently doing a mix of both.

Reply:
The combination occurs when one of the parameters with a default value is not assigned a value. By leaving the default value, Android needs to know the next parameter we are configuring. That’s why the compiler automatically adds the parameter name.

Reply:
Yeah I understand that piece, was moreso curious if we have decided on a general style on the Android side for Kotlin features since looking through the SDK it's a bit all over the place. The following is also valid syntax:
```kotlin
val event = AnalyticsEvent(
    name = eventName,
    payPalContextId = params.payPalContextId,
    linkType = params.linkType,
    venmoInstalled = isVenmoInstalled,
    isVaultRequest = params.isVaultRequest,
    startTime = params.startTime,
    endTime = params.endTime,
    endpoint = params.endpoint
)
```
This isn't a blocker and may not be decided by the Android folks yet so just wanted to bring it up. It does not seem like our pattern is to only used when assigning a value, we are instead kinda all over the place. 🙈

Reply:
Yes, thanks for the feedback. I'd like to know what the rest of the team thinks about all of this